src/Api/SecretsManager/Models/Response/GrantedProjectAccessPolicyPermissionDetailsResponseModel.cs

```diff
@@ -4,22 +4,22 @@
 
 namespace Bit.Api.SecretsManager.Models.Response;
 
-public class ServiceAccountProjectAccessPolicyPermissionDetailsResponseModel : ResponseModel
+public class GrantedProjectAccessPolicyPermissionDetailsResponseModel : ResponseModel
 {
     private const string _objectName = "serviceAccountProjectAccessPolicyPermissionDetails";
 
-    public ServiceAccountProjectAccessPolicyPermissionDetailsResponseModel(
+    public GrantedProjectAccessPolicyPermissionDetailsResponseModel(
```

Comment:
update `_objectName`

Reply:
Great catch, updated in https://github.com/bitwarden/server/pull/4146/commits/cc7f5dd85e2e29873417c53e5c134685c0f27aa3